sql/telemetry_derived/devtools_accessiblility_panel_usage_v1/query.sql

```diff
@@ -0,0 +1,172 @@
+  SAFE_DIVIDE(SUM(node_inspected_count), COUNT(*)) AS node_inspected_per_client_day,
+  COUNTIF(node_inspected_count > 0) AS node_inspected_users,
+  COUNTIF(picker_used_count > 0) AS picker_users,
+  COUNTIF(picker_used_count > 0) / (COUNT(*)) AS picker_users_per_client_day,
+  SAFE_DIVIDE(SUM(picker_used_count), COUNT(*)) AS picker_count_per_client_day,
+  SAFE_DIVIDE(SUM(browser_context_count), COUNT(*)) AS browser_context_per_client_day,
+  COUNTIF(browser_context_count > 0) AS browser_context_users,
+  SAFE_DIVIDE(SUM(inspector_context_count), COUNT(*)) AS inspector_context_per_client_day,
+  COUNTIF(inspector_context_count > 0) AS inspector_context_users,
+  COUNTIF(service_enabled_count > 0) AS service_enabled_users,
+  SAFE_DIVIDE(COUNTIF(service_enabled_count > 0), COUNT(*)) AS service_enabled_users_per_client_day,
+  SAFE_DIVIDE(SUM(audit_all_count), COUNT(*)) AS audit_all_per_client_day,
+  COUNTIF(audit_all_count > 0) AS audit_all_users,
+  SAFE_DIVIDE(SUM(audit_contrast_count), COUNT(*)) AS audit_contrast_per_client_day,
+  COUNTIF(audit_contrast_count > 0) AS audit_contrast_users,
+  SAFE_DIVIDE(SUM(audit_keyboard_count), COUNT(*)) AS audit_keyboard_per_client_day,
+  COUNTIF(audit_keyboard_count > 0) AS audit_keyboard_users,
+  SAFE_DIVIDE(SUM(audit_text_label_count), COUNT(*)) AS audit_text_label_per_client_day,
+  COUNTIF(audit_text_label_count > 0) AS audit_text_label_users,
+  SAFE_DIVIDE(
+    SUM(simulation_deuteranomaly_count),
+    COUNT(*)
+  ) AS simulation_deuteranomaly_per_client_day,
+  COUNTIF(simulation_deuteranomaly_count > 0) AS simulation_deuteranomaly_users,
+  SAFE_DIVIDE(SUM(simulation_protanomaly_count), COUNT(*)) AS simulation_protanomaly_per_client_day,
+  COUNTIF(simulation_protanomaly_count > 0) AS simulation_protanomaly_users,
+  SAFE_DIVIDE(SUM(simulation_protanopia_count), COUNT(*)) AS simulation_protanopia_per_client_day,
+  COUNTIF(simulation_protanopia_count > 0) AS simulation_protanopia_users,
+  SAFE_DIVIDE(
+    SUM(simulation_deuteranopia_count),
+    COUNT(*)
+  ) AS simulation_deuteranopia_per_client_day,
+  COUNTIF(simulation_deuteranopia_count > 0) AS simulation_deuteranopia_users,
+  SAFE_DIVIDE(SUM(simulation_tritanopia_count), COUNT(*)) AS simulation_tritanopia_per_client_day,
+  COUNTIF(simulation_tritanopia_count > 0) AS simulation_tritanopia_users,
+  SAFE_DIVIDE(SUM(simulation_tritanomaly_count), COUNT(*)) AS simulation_tritanomaly_per_client_day,
+  COUNTIF(simulation_tritanomaly_count > 0) AS simulation_tritanomaly_users,
+  SAFE_DIVIDE(
+    SUM(simulation_contrast_loss_count),
+    COUNT(*)
+  ) AS simulation_contrast_loss_per_client_day,
+  COUNTIF(simulation_contrast_loss_count > 0) AS simulation_contrast_loss_users,
+  SUM(
+    simulation_deuteranomaly_count + simulation_protanomaly_count + simulation_protanopia_count + simulation_deuteranopia_count + simulation_tritanopia_count + simulation_tritanomaly_count + simulation_contrast_loss_count
```

Comment:
@yzen just a note, it looks like this always 0. is that expected? https://sql.telemetry.mozilla.org/queries/73089#183070

Reply:
yeah, I think this query is incorrect, I did not get to fixing it just yet..

Reply:
It would be nice to fix this before we merge, so we don't have to backfill twice.

I think the actual SUM part is fine, it's just we're not getting any values for the keyed scalars for... some reason? I notice that `simulation_<x>_count` is always 0.

Here's the definition in the probe dictionary if you haven't seen it before: https://probes.telemetry.mozilla.org/?search=devtools&optout=true&view=detail&probeId=scalar%2Fdevtools.accessibility.simulation_activated

Reply:
Turns out I was using incorrect keys for the scalar (I also needed to remove some keys that are no longer present and added a new one). They should be: `PROTANOPIA`, `DEUTERANOPIA`, `TRITANOPIA`, `ACHROMATOPSIA` and `CONTRAST_LOSS`. In the original query, the simulation related code looks like:
```
    udf.get_key(payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated, "PROTANOPIA") * 100 AS simulation_protanopia_count,
    udf.get_key(payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated, "DEUTERANOPIA") * 100 AS simulation_deuteranopia_count,
    udf.get_key(payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated, "TRITANOPIA") * 100 AS simulation_tritanopia_count,
    udf.get_key(payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated, "ACHROMATOPSIA") * 100 AS simulation_achromatopsia_count,
    udf.get_key(payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated, "CONTRAST_LOSS") * 100 AS simulation_contrast_loss_count
```
And in the second query I changed simulation related code to:
```
    SUM(simulation_protanopia_count) / (COUNT(*) * 100) AS simulation_protanopia_per_client_day,
    COALESCE(SUM(CAST(simulation_protanopia_count > 0 AS INT64)), 0) * 100 AS simulation_protanopia_users,
    SUM(simulation_deuteranopia_count) / (COUNT(*) * 100) AS simulation_deuteranopia_per_client_day,
    COALESCE(SUM(CAST(simulation_deuteranopia_count > 0 AS INT64)), 0) * 100 AS simulation_deuteranopia_users,
    SUM(simulation_tritanopia_count) / (COUNT(*) * 100) AS simulation_tritanopia_per_client_day,
    COALESCE(SUM(CAST(simulation_tritanopia_count > 0 AS INT64)), 0) * 100 AS simulation_tritanopia_users,
    SUM(simulation_achromatopsia_count) / (COUNT(*) * 100) AS simulation_achromatopsia_per_client_day,
    COALESCE(SUM(CAST(simulation_achromatopsia_count > 0 AS INT64)), 0) * 100 AS simulation_achromatopsia_users,
    SUM(simulation_contrast_loss_count) / (COUNT(*) * 100) AS simulation_contrast_loss_per_client_day,
    COALESCE(SUM(CAST(simulation_contrast_loss_count > 0 AS INT64)), 0) * 100 AS simulation_contrast_loss_users,
    
    SUM(simulation_protanopia_count + simulation_deuteranopia_count + simulation_tritanopia_count + simulation_achromatopsia_count + simulation_contrast_loss_count) AS simulations
```

Reply:
Is it just me or is the total number of simulations still equalling 0 in your query? https://sql.telemetry.mozilla.org/queries/73089/source

Reply:
Yeah, this functionality depends on web render so if there would be very few users at the moment. There's also sampling. I tried without sampling and I got some numbers on non-release channels but the are very low (<10 total)

Reply:
@yzen and I did some analysis on Slack. It looks like we get a fairly small set of results if we turn off sampling. Given that one month's worth of this query is only half a TB in size, we think it's worth turning off sampling so we can detect small changes in things like this.